Makefile

```diff
@@ -44,7 +44,6 @@ build-docs:
 	$(MAKE) -C docs doctest
 
 docs: build-docs validate-docs
-	open docs/_build/html/index.html
```

Comment:
Not sure why this is an issue in your local environment but it works on my end. I don't see a need to remove this.

Reply:
If you run `make docs` locally, you don't see the very last lines of output as: 
```
open docs/_build/html/index.html
make: open: Command not found
make: *** [Makefile:47: docs] Error 127
```

Reply:
Meh, it's not there in the CI `docs` run. Un-deleted.